### PR TITLE
feat-add alert component to docs

### PR DESCRIPTION
**Pull request by mcmonojit**
**Added alert component page to documentation**
- simple alert
- alert with close button

For preview edit the URL section `github` to `githubbox` .

### CODE REVIEW

File: css/components/alert.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    position: relative;
+    top:0;
+    right: 0;
+}
+
+/* .btn-right{
+    display: flex;
+    justify-content: space-between;
+} */
+
```
Comment (Rockyraven) [inline css/components/alert.css]: remove unwanted code

File: css/components/alert.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.alertClose h4{
+    margin: 0;
+    /* display: inline; */
+}
+
+.alertClose p{
```
Comment (Rockyraven) [inline css/components/alert.css:71]: remove too

File: docs/docs.css [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+html{
+    background-color: var(--zinc);
+}
+
+body{
+    background-color: white;
+    font-family: 'Quicksand', sans-serif;
+    /* font-family: 'Poppins', sans-serif; */
+    margin: 0px;
+    height: 100vh;
```
Comment (Rockyraven) [inline docs/docs.css:38]: remove here also

**Review by Rockyraven — COMMENTED** [review]
you have use many times unwanted code 
remove all unwanted code

File: css/components/alert.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    margin: 0;
+    margin-top: 0.5rem;
+}
+
+.close-btn:hover{
+    background-color: #fdba74;
```
Comment (AtulHande03a) [inline css/components/alert.css]: use variables for colors

**Review by AtulHande03a — APPROVED** [review]
use of variables for color and to remove any unwanted comments .. rest all looks good

File: docs/docs.css (hunk 3 above)
Comment (mcmonojit) [reply docs/docs.css:38]: Implemented all the recomended changes in the commit #[c1c7865](https://github.com/mcmonojit/CoolWaterUI/pull/1/commits/c1c7865a9062a8223606e1d8ed9b9d4f9c688474)

File: css/components/alert.css (hunk 4 above)
Comment (mcmonojit) [reply css/components/alert.css]: Implemented all the recomended changes in the commit #[c1c7865](https://github.com/mcmonojit/CoolWaterUI/pull/1/commits/c1c7865a9062a8223606e1d8ed9b9d4f9c688474)

**Review by Rockyraven — APPROVED** [review]
All good to me you can merge

**Pull request merged**